pkg/sdk/renderer/argo/renderer_test.go

```diff
@@ -277,22 +260,35 @@ func TestRenderHappyPathWithCustomPolicies(t *testing.T) {
 			typeInstanceHandler := NewTypeInstanceHandler("alpine:3.7")
 			typeInstanceHandler.SetGenUUID(genUUID)
 
+			interfaceIOValidator := actionvalidation.NewValidator(fakeCli)
```

Comment:
Theoretically I could mock PolicyEnforcedClient or PolicyEnforcedClient's validator here as well, similarly as we did it before with Workflow Validator, but that would be more error-prone and difficult to test different cases. And it wouldn't bring much benefit as we would mostly test the mock.
IMHO PolicyEnforcedClient is one of crucial parts of rendering process, that's why I wanted to test it with renderer and validation in place. We use fileSystem Hub Client, and the only missing part we didn't have, were the Types imported for proper validation.

And I believe that was good idea - it turned out that our renderer test cases didn't make sense most of the time 😄 I fixed them.